common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -35,7 +35,7 @@
                         '[data-discussion-id="' + this.getCurrentTopicId() + '"]'
                     ));
                 } else if ($general.length > 0) {
-                    this.setTopic($general);
+                    this.setTopic($general.first());
```

Comment:
This fixed the bug I was seeing with the group dropdown enablement state. If more than 1 topic had "General" in it, $general would have more than one item. Then the code that set the enablement state on the selector could end up getting the wrong "divided" attribute.

Seeing how this is quite an edge case, I don't think it is worth a test.